src/sankey/index.js

```diff
@@ -0,0 +1,137 @@
+import React, {PropTypes, Component} from 'react';
+import {voronoi} from 'd3-voronoi';
+import {sankey} from 'd3-sankey-align';
+
+import {DISCRETE_COLOR_RANGE} from 'theme';
+
+const noop = f => f;
```

Comment:
Why exactly? Almost every variable in our code is a constant, but we're using normal camelCase, why not stay consistent?